src/widgets/splits/SplitInput.cpp

```diff
@@ -217,6 +217,8 @@ void SplitInput::initLayout()
         app->getFonts()->fontChanged, [=, this]() {
             this->ui_.textEdit->setFont(
                 app->getFonts()->getFont(FontStyle::ChatMedium, this->scale()));
+            this->ui_.textEditLength->setFont(app->getFonts()->getFont(
+                FontStyle::TimestampMedium, this->scale()));
             this->ui_.replyLabel->setFont(app->getFonts()->getFont(
                 FontStyle::ChatMediumBold, this->scale()));
```

Comment:
This is basically the same code as in `scaleChangedEvent`. We should probably have some `updateFonts` method to deduplicate.

Reply:
sure. I might make another PR after this one gets merged.